src/Schema/Event.ts

```diff
@@ -93,3 +105,37 @@ export interface SuccessfullyLoggedIn {
   auth_redirect: string
   service: AuthService
 }
+
+/**
+ * A user taps a rail, card, or other grouping of entities on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedEntityGroup"
+ *    context_module: "newWorksByArtistsYouFollowRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Artwork
+ *    destination_screen_owner_id: "5e9a7a238483bf000e2c4c5e"
+ *    destination_screen_owner_slug: "romain-jacquet-lagreze-makeshift-garden-hong-kong"
+ *    horizontal_slide_position: 1
+ *    module_height: "single"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedEntityGroup {
+  action: ActionType.tappedEntityGroup
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
```

Comment:
does `?:` mean optional?

Reply:
Yes, that's right. Find more info on interfaces here: https://www.typescriptlang.org/docs/handbook/interfaces.html